scripts/Hello_World_Klotzey13.py

```diff
@@ -0,0 +1 @@
+print("Hello World!!!")
```

Comment:
Please add comments to the file which contain the required info. You will find which info you need in issue #162.